libs/filameshio/src/MeshReader.cpp

```diff
@@ -217,14 +219,14 @@ MeshReader::Mesh MeshReader::loadMeshFromBuffer(filament::Engine* engine,
     Part* parts = (Part*) p;
     p += header->parts * sizeof(Part);
 
-    uint32_t materialCount = (uint32_t) *p;
+    const uint32_t materialCount = *(const uint32_t *)p;
```

Comment:
`uint32_t const materialCount = *(uint32_t const *)p;`

Reply:
I know, but that's the style we strive use elsewhere (even if we're not always consistant, that's the style we should be using).

Reply:
OK. It's a choice.